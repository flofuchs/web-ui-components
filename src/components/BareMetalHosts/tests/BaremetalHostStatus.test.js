import React from 'react';
import { shallow } from 'enzyme';

import { BaremetalHostStatus } from '..';

import BaremetalHostStatusFixture from '../fixtures/BaremetalHostStatus.fixture';

const testBaremetalHostStatus = () => <BaremetalHostStatus {...BaremetalHostStatusFixture[0].props} />;
const testBaremetalHostStatusSuccess = () => <BaremetalHostStatus {...BaremetalHostStatusFixture[1].props} />;

describe('<BaremetalHostStatus />', () => {
  it('renders a generic status message', () => {
    const component = shallow(testBaremetalHostStatus());
    expect(component).toMatchSnapshot();
  });
  it('renders a generic success message', () => {
    const component = shallow(testBaremetalHostStatusSuccess());
    expect(component).toMatchSnapshot();
  });
});
