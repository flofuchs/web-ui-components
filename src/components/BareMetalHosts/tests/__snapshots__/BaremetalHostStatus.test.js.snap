// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaremetalHostStatus /> renders a generic status message 1`] = `
<GenericStatus
  status="unknown state"
  text="unknown state"
/>
`;

exports[`<BaremetalHostStatus /> renders a generic success message 1`] = `
<GenericSuccess
  status="provisioned"
  text="Provisioned"
/>
`;
